mocks/mock-data/disputes.ts

```diff
@@ -7,6 +7,200 @@ const getDate = (daysOffset = 0, originDate = new Date()) => {
     return date.toISOString();
 };
 
+const DEFAULT_DETAIL_DEFENSE: IDisputeDetail['defense'] = {
+    defendedOn: getDate(-1),
+    defendedThroughComponent: true,
+    reason: 'Dispute reason',
+};
+const DEFAULT_DETAIL_PAYMENT: IDisputeDetail['payment'] = {
+    balanceAccount: { timeZone: 'UTC', description: 'Main balance account' },
+    isRefunded: false,
+    merchantReference: 'b2c3d4e5-f6g7-5890-efgh',
+    paymentMethod: { lastFourDigits: '1234', type: 'visa' },
+    pspReference: 'a1b2c3d4-e5f6-4789-abcd',
+};
+
+const DEFAULT_DETAIL_DISPUTE: IDisputeDetail['dispute'] = {
+    amount: {
+        value: 211100,
+        currency: 'EUR',
+    },
+    dueDate: new Date(new Date().setHours(23)).toISOString(),
+    createdAt: getDate(-10),
+    pspReference: 'a1b2c3d4-e5f6-4789-abcd-000000000001',
+    reason: {
+        category: 'REQUEST_FOR_INFORMATION',
+        title: 'Request for information',
+        code: '1234',
+    },
+    allowedDefenseReasons: [],
+    defensibility: 'DEFENDABLE',
+    type: 'CHARGEBACK',
+    status: 'UNDEFENDED',
+};
+
+const DEFAULT_DISPUTE_DETAIL: IDisputeDetail = {
+    dispute: DEFAULT_DETAIL_DISPUTE,
+    defense: DEFAULT_DETAIL_DEFENSE,
+    payment: DEFAULT_DETAIL_PAYMENT,
+};
+
+// CHARGEBACKS
+
+export const CHARGEBACK_UNDEFENDED: IDisputeDetail = {
+    ...DEFAULT_DISPUTE_DETAIL,
+    dispute: {
+        ...DEFAULT_DISPUTE_DETAIL.dispute,
+        status: 'UNDEFENDED',
+    },
```

Comment:
This will include the `defense` field, which is not applicable in this case. What do you think about doing this instead?

```suggestion
    dispute: {
        ...DEFAULT_DETAIL_DISPUTE,
        status: 'UNDEFENDED',
    },
    payment: DEFAULT_DETAIL_PAYMENT,
```

Reply:
A similar change is applicable in a couple other places.

Reply:
✅